airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json

```diff
@@ -76,14 +76,23 @@
           "enum": ["UNKNOWN", "ENABLED", "CANCELED", "SUSPENDED", "CLOSED"]
         }
       },
+      "manager_id": {
```

Comment:
WDYT about doing this in a separate PR so we can minimize the changes that we introduce when adding concurrency?